
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1265067049 -3600
# Node ID 00c8a9985e2035ae6dce224d309fc0be25da6d8f
# Parent  f23c5a878fb1581d2be9c19975307c2c8d463b61
libv4l: Update sonixb device table entries

From: Hans de Goede <hdegoede@redhat.com>

Use a default gamma of 1500 for all sonixb cams, this gives a much better
picture; and enable software whitebalance be default for all sonixb cams,
except for those with ov sensors.

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Mon Feb 01 13:32:46 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Tue Feb 02 00:30:49 2010 +0100
@@ -285,9 +285,13 @@
 
 /* Second: devices which should use some software processing by default */
   /* sn9c101 / sn9c102 based devices (sonixb) */
-  { 0x0c45, 0x6001, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110C */
-  { 0x0c45, 0x6005, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110C */
-  { 0x0c45, 0x6007, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110D */
+  { 0x0c45, 0x6011, 0,    NULL, NULL, 0, 1500 }, /* OV6650, no WB needed */
+  { 0x0c45, 0x6019, 0,    NULL, NULL, 0, 1500 }, /* OV7630, no WB needed */
+  { 0x0c45, 0x608f, 0,    NULL, NULL, 0, 1500 }, /* OV7630, no WB needed */
+  { 0x0c45, 0x60b0, 0,    NULL, NULL, 0, 1500 }, /* OV7630, no WB needed */
+  { 0x0c45, 0x6000, 0x1f, NULL, NULL, V4LCONTROL_WANTS_WB, 1500 }, /* other */
+  { 0x0c45, 0x6020, 0x0f, NULL, NULL, V4LCONTROL_WANTS_WB, 1500 }, /* other */
+  { 0x0c45, 0x60af, 0,    NULL, NULL, V4LCONTROL_WANTS_WB, 1500 }, /* PAS202 */
   /* sn9c105 / sn9c120 based devices (sonixj) */
   { 0x0c45, 0x60fe, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */
   { 0x0c45, 0x610e, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */

